scripts/script.js

```diff
@@ -0,0 +1,21 @@
+
+window.onload=function(){
+    document.getElementById("menu-hamburger").addEventListener("click", toggleMenu);
+    window.addEventListener("resize", function(event) {
+        if (navBar.classList.contains("nav-mobile")) {
+            navBar.classList.remove("nav-mobile");
+            navBar.classList.add("navigation");
+        }
+}
+
+const toggleMenu = () => {
+    var navBar=document.getElementById("navigation");
+    if (navBar.classList.contains("nav-mobile")) {
+        navBar.classList.remove("nav-mobile");
+        navBar.classList.add("navigation");
+    } else {
+        navBar.classList.remove("navigation");
+        navBar.classList.add("nav-mobile");
+    }
+})
+}
```

Comment:
Now the resize code references a 'navBar' variable which is not defined, also there are syntax errors in this code.

Do you have an automatic linter in your editor? The linter/intellisense should tell you about these problems, see how to enable it for vs code here: https://code.visualstudio.com/docs/languages/javascript

It also looks like all the line endings or spacing have been changed, and the PR is highlighting everything as a change, which makes further code reviews hard... can you look into it?

Reply:
Hi Alberto, 

Not sure what happened there with line endings, but after 2 hours of hair pulling I finally figured out how to remove commits. Should be fine now and resolved the JS issue as well. 

Appreciate your patience with this